CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.10.1 - 2017-04-11
+
+- Update fury to 3.0.0-beta.1
```

Comment:
What do you mean by that? The output or the API is not getting changed at all. Only the internals are.